### DISCUSSION TITLE
A way to quickly visit the oldest starred repos

### DISCUSSION THREAD

**Discussion by sayanarijit** [community/community#12924] [Other Feature Feedback, Questions, & Ideas]
Hi, I see we can go to "next" and "previous" page when browsing through the starred repos, but couldn't find a way to visit the "last" page.

The sort function doesn't include a "reverse" option, either.

I need this feature as my interests have shifted, and I want to "unstar" the repos I'm not interested in anymore.

**Comment by StreetStrider** [comment]
Same scenario for me. I must note that it was possible with the old-style pagination API (providing offset) like year or two ago. I was able to pick the oldest page manually by trial and error and move from there to the modern pages.

**Comment by StreetStrider** [comment]
I still need this. I have a lot of starred projects and sometimes I want to review the oldest ones and un-star nowaday's irrelevant ones. This was totally possible with the old pagination model. Today it is not possible, since you can review stars only from the most actual to least actual.

The solution would be to:
1. return pagination ui
2. allow to sort by oldest starred, by oldest active and by least stars (or via reverse mode for the existing sorts)
3. implement both 1 and 2
